src/main/java/pl/tomaszdziurko/jvm_bloggers/metadata/Metadata.java

```diff
@@ -29,7 +29,7 @@
     @Column(name = "value", nullable = false, length = 2500)
     private String value;
 
-    public Setting(String name, String value) {
+    public Metadata(String name, String value) {
```

Comment:
You can remove the constructor, it will be generated by already added Lomboks [`@Data`](https://projectlombok.org/features/Data.html)  class level annotation.